### PR TITLE
Sync play status with HBO GO (EU handler)

### PR DESCRIPTION
**Pull request by sile70000**
## Please check that your pull request pass this checks:

-  [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
-  [x] My code is both Python 2 and 3 compatible
-  [x] My code follows the code style of this project
-  [x] I made sure there wasn't another Pull Request opened for the same update/change
-  [x] I have successfully tested my changes locally

## Types of changes

-  [ ] Bug fix (non-breaking change which fixes an issue)
-  [x] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to change)
-  [ ] Adding a new region/hbo go version

## Description:
Sync resume point with Hbo.go

### CODE REVIEW

File: hbogolib/handlereu.py [hunk 1]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+        IsPlaying = xbmc.Player().isPlayingVideo()
+
+        while not IsPlaying:
+            IsPlaying = xbmc.Player().isPlayingVideo()
+            xbmc.sleep(300)
+
+        while (IsPlaying and ExternalId == xbmc.Player().getVideoInfoTag().getTagLine()) :
+            InfoTag = xbmc.Player().getVideoInfoTag()
+            ExternalId = InfoTag.getTagLine()
+            MediaType=InfoTag.getMediaType()
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Exactly one space required around assignment](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 2]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+    def get_elapsed(self, ExternalId):
+        Position = 0
+        for list in self.JsonHis:
+            if list['externalId'] == ExternalId:
+                Position = list['position']
+        xbmc.log('ExternalId/position '+str(ExternalId)+'/'+str(Position), xbmc.LOGNOTICE)
+        return Position
+
+    def update_history(self, ExternalId, MediaType, Current_Time, Percent_Elapsed):
+        if (MediaType == 'movie' ):
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [No space allowed before bracket](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 17
           

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 3]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+            MediaType = '1'
+        elif (MediaType == 'episode' ):
+            MediaType = '3'
+        resume_payload = '{"CustomerId":"'+self.GOcustomerId+'","CountryCode":"'+self.LANGUAGE_CODE + '","ExternalId":"'+ExternalId+'","ContentType":'+MediaType +',"Position":'+Current_Time+',"ElapsedPercentage":'+Percent_Elapsed +',"LoginSessionId":"'+str(self.sessionId)+'"}'
+        history_headers=self.loggedin_headers
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Exactly one space required around assignment](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 4]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+        for list in self.JsonHis:
+            if list['externalId'] == ExternalId:
+                Position = list['position']
+        xbmc.log('ExternalId/position '+str(ExternalId)+'/'+str(Position), xbmc.LOGNOTICE)
+        return Position
+
+    def update_history(self, ExternalId, MediaType, Current_Time, Percent_Elapsed):
+        if (MediaType == 'movie' ):
+            MediaType = '1'
+        elif (MediaType == 'episode' ):
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [No space allowed before bracket](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 5]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+
+        while (IsPlaying and ExternalId == xbmc.Player().getVideoInfoTag().getTagLine()) :
+            InfoTag = xbmc.Player().getVideoInfoTag()
+            ExternalId = InfoTag.getTagLine()
+            MediaType=InfoTag.getMediaType()
+            Current_Time = int(xbmc.Player().getTime())
+            Total_Time = int(xbmc.Player().getTotalTime())
+            xbmc.sleep(300)
+            IsPlaying = xbmc.Player().isPlayingVideo()
+        else:
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Else clause on loop without a break statement](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 6]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+                Position = list['position']
+        xbmc.log('ExternalId/position '+str(ExternalId)+'/'+str(Position), xbmc.LOGNOTICE)
+        return Position
+
+    def update_history(self, ExternalId, MediaType, Current_Time, Percent_Elapsed):
+        if (MediaType == 'movie' ):
+            MediaType = '1'
+        elif (MediaType == 'episode' ):
+            MediaType = '3'
+        resume_payload = '{"CustomerId":"'+self.GOcustomerId+'","CountryCode":"'+self.LANGUAGE_CODE + '","ExternalId":"'+ExternalId+'","ContentType":'+MediaType +',"Position":'+Current_Time+',"ElapsedPercentage":'+Percent_Elapsed +',"LoginSessionId":"'+str(self.sessionId)+'"}'
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [line too long (277 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 7]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+        history_headers=self.loggedin_headers
+        history_headers['Content-Type'] = 'application/json'
+        self.post_to_hbogo(self.API_URL_HIS, history_headers, resume_payload,'')
+		
+    def track_elapsed(self,TagLine):
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Exactly one space required after comma](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 8]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
         liz.setInfo(type="Video", infoLabels={"Title": name})
         liz.setProperty('isPlayable', "false")
         xbmcplugin.addDirectoryItem(handle=self.handle, url=category_url, listitem=liz, isFolder=True)
+
+    def get_elapsed(self, ExternalId):
+        Position = 0
+        for list in self.JsonHis:
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Redefining built-in 'list'](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 9]
```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
...
+
+    def update_history(self, ExternalId, MediaType, Current_Time, Percent_Elapsed):
+        if (MediaType == 'movie' ):
+            MediaType = '1'
+        elif (MediaType == 'episode' ):
+            MediaType = '3'
+        resume_payload = '{"CustomerId":"'+self.GOcustomerId+'","CountryCode":"'+self.LANGUAGE_CODE + '","ExternalId":"'+ExternalId+'","ContentType":'+MediaType +',"Position":'+Current_Time+',"ElapsedPercentage":'+Percent_Elapsed +',"LoginSessionId":"'+str(self.sessionId)+'"}'
+        history_headers=self.loggedin_headers
+        history_headers['Content-Type'] = 'application/json'
+        self.post_to_hbogo(self.API_URL_HIS, history_headers, resume_payload,'')
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Exactly one space required after comma](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 10]
```diff
@@ -1294,3 +1309,47 @@ def addCat(self, name, url, icon, mode):
...
+                Position = listIds['position']
+        xbmc.log('ExternalId/position '+str(ExternalId)+'/'+str(Position), xbmc.LOGNOTICE)
+        return Position
+
+    def update_history(self, ExternalId, MediaType, Current_Time, Percent_Elapsed):
+        if (MediaType == 'movie'):
+            MediaType = '1'
+        elif (MediaType == 'episode'):
+            MediaType = '3'
+        resume_payload = '{"CustomerId":"'+self.GOcustomerId+'","CountryCode":"'+self.LANGUAGE_CODE + '","ExternalId":"'+ExternalId+'","ContentType":'+MediaType +',"Position":'+Current_Time+',"ElapsedPercentage":'+Percent_Elapsed +',"LoginSessionId":"'+str(self.sessionId)+'"}'
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [line too long (277 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 11]
```diff
@@ -1294,3 +1309,47 @@ def addCat(self, name, url, icon, mode):
...
+
+        while (IsPlaying and ExternalId == xbmc.Player().getVideoInfoTag().getTagLine()) :
+            InfoTag = xbmc.Player().getVideoInfoTag()
+            ExternalId = InfoTag.getTagLine()
+            MediaType = InfoTag.getMediaType()
+            Current_Time = int(xbmc.Player().getTime())
+            Total_Time = int(xbmc.Player().getTotalTime())
+            xbmc.sleep(300)
+            IsPlaying = xbmc.Player().isPlayingVideo()
+        else:
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Else clause on loop without a break statement](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 12]
```diff
@@ -1294,3 +1309,49 @@ def addCat(self, name, url, icon, mode):
...
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Else clause on loop without a break statement](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 13]
```diff
@@ -1294,3 +1309,48 @@ def addCat(self, name, url, icon, mode):
...
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Else clause on loop without a break statement](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Hello nice work @sile70000. Haven't have time to review in detail but from a quick test, all look fine any reason you closed this?
Just the track_elapsed function can be improved and a fails safe added in case of playback never starts.

**Comment by sile70000** [comment]
Hi,I don't know much about Python or coding so the loop at the end have some issues:When you Context Menu --> Play from here• Sometimes you get divide by zero -- meaning the total time is not fetched• Sometimes the last played file is not synced too Hbo.go. This I don't know why.I've closed it because I've got away and I couldn't look at it.I will reopen it now and I'll test it some more.If you have some time please take a look at it. 
Thanks.
    On Sunday, December 15, 2019, 05:35:00 PM GMT+2, Arv.Void <notifications@github.com> wrote:  
 
 
Reopened #88.

**Comment by arvvoid** [comment]
@sile70000 Don't worry there is no rush, I will look into it as soon as I have some free time. The track_elapsed function needs improvement, tomorrow probably I will have some spare time will see.

**Comment by arvvoid** [comment]
@sile70000 I fixed and optimized most of the code, have to improve the set watched status a bit. Then will do some additional improvements after merging (option to disable ecc).

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

**Comment by arvvoid** [comment]
Since it relies on data set in the list item this will not work if playback start from an external source like exported strm file, so will have to make some modification to make it more robust before merging.

File: hbogolib/handlereu.py [hunk 14]
```diff
@@ -1228,12 +1275,17 @@ def addLink(self, title, mode):
...
                     })
         liz.addStreamInfo('video', {'width': 1920, 'height': 1080})
         liz.addStreamInfo('video', {'aspect': 1.78, 'codec': 'h264'})
         liz.addStreamInfo('audio', {'codec': 'aac', 'channels': 2})
         liz.setProperty("IsPlayable", "true")
+        if hbogo_position > -1:
+            liz.setProperty("totaltime", str(title['Duration']))
+            liz.setProperty("resumetime", str(hbogo_position))
+            if int(int(hbogo_position) / int(title['Duration'])*100) > 89: #set as watched if 90% is watched
+                liz.setInfo(type="Video",infoLabels={"overlay": "6"})
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Exactly one space required after comma](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Clones added
============
- hbogolib/handlereu.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 15]
```diff
@@ -1058,6 +1066,42 @@ def play(self, url, content_id):
...
+            if self.force_original_names:
+                name = py2_encode(title['OriginalName'])
+            plot = py2_encode(title['Abstract'])
+            if 'Description' in title:
+                if title['Description'] is not None:
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                    infoLabels={
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong continued indentation (add 6 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           

Clones added
============
- hbogolib/handlereu.py  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

File: hbogolib/handlereu.py [hunk 16]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
+                                        "cast": [title['Cast'].split(', ')][0], "director": title['Director'],
+                                        "writer": title['Writer'], "duration": title['Duration'], "genre": title['Genre'],
+                                        "title": name, "originaltitle": title['OriginalName'],
+                                        "year": title['ProductionYear'],
+                                        "TagLine": externalid
+                                     }
+                          )
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong continued indentation (remove 1 space).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 17]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
+                                        "cast": [title['Cast'].split(', ')][0], "director": title['Director'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 18]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                                        "writer": title['Writer'], "duration": title['Duration'], "genre": title['Genre'],
+                                        "title": name, "originaltitle": title['OriginalName'],
+                                        "year": title['ProductionYear'],
+                                        "TagLine": externalid
+                                     }
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation.](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 19]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+            plot = py2_encode(title['Abstract'])
+            if 'Description' in title:
+                if title['Description'] is not None:
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
+                                        "season": title['Tracking']['SeasonNumber'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 20]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
+                                        "cast": [title['Cast'].split(', ')][0], "director": title['Director'],
+                                        "writer": title['Writer'], "duration": title['Duration'], "genre": title['Genre'],
+                                        "title": name, "originaltitle": title['OriginalName'],
+                                        "year": title['ProductionYear'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 21]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                                        "TagLine": externalid
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 22]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
+                                        "cast": [title['Cast'].split(', ')][0], "director": title['Director'],
+                                        "writer": title['Writer'], "duration": title['Duration'], "genre": title['Genre'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 23]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+            if 'Description' in title:
+                if title['Description'] is not None:
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 24]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                name = py2_encode(title['OriginalName'])
+            plot = py2_encode(title['Abstract'])
+            if 'Description' in title:
+                if title['Description'] is not None:
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 25]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
+                                        "cast": [title['Cast'].split(', ')][0], "director": title['Director'],
+                                        "writer": title['Writer'], "duration": title['Duration'], "genre": title['Genre'],
+                                        "title": name, "originaltitle": title['OriginalName'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

File: hbogolib/handlereu.py [hunk 26]
```diff
@@ -1058,6 +1066,43 @@ def play(self, url, content_id):
...
+                if title['Description'] is not None:
+                    plot = py2_encode(title['Description'])
+            if 'AvailabilityTo' in title:
+                plot = plot + ' ' + self.LB_EPISODE_UNTILL + ' ' + py2_encode(title['AvailabilityTo'])
+        list_item.setInfo(type="Video",
+                          infoLabels={
+                                        "mediatype": media_type, "episode": title['Tracking']['EpisodeNumber'],
+                                        "season": title['Tracking']['SeasonNumber'],
+                                        "tvshowtitle": title['Tracking']['ShowName'], "plot": plot,
+                                        "mpaa": str(title['AgeRating']) + '+', "rating": title['ImdbRate'],
```
Comment (arvvoid) [inline hbogolib/handlereu.py]: Codacy Issue found: [Wrong hanging indentation (remove 10 spaces).](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- hbogolib/handlereu.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960&bid=15650497)

**Comment by arvvoid** [comment]
@all-contributors please add @sile70000 for code

**Comment by arvvoid** [comment]
This now is solid, I will merge it in a separate branch and make some fine-tuning before making a test release.

**Pull request merged**